device/core/src/pnp/json.ts

```diff
@@ -5,3 +5,5 @@ export type JSONValue =
  | null
  | JSONValue[]
  | {[key: string]: JSONValue};
+
+export type JSONObject = {[key: string]: JSONValue};
```

Comment:
>JSONObject [](http://example.com/codeflow?start=12&length=10)

super nit-picky semantic comment.  Technically, These are not JSON anything.  JSON is the text representation of native objects.  You serialize objects of these types into JSON, but you could also serialize them into other forms.  I don't know what a better name would be.  Are they PnpCompatibleObjects and PnpCompatibleValues?  (I don't rememer if we can expose "Pnp" in type names or not.)

Reply:
Changed to `JSONSerializableObject` and `JSONSerializableValue`